streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -320,7 +318,7 @@ private void commitOffsets(final boolean startNewTransaction) {
                 try {
                     consumer.commitSync(consumedOffsetsAndMetadata);
                 } catch (final CommitFailedException e) {
-                    log.warn("{} Failed offset commits {}: ", logPrefix, consumedOffsetsAndMetadata, e);
+                    log.warn("{} Failed offset commits {} due to CommitFailedException", logPrefix, consumedOffsetsAndMetadata);
```

Comment:
We do not print the error any more in this WARN entry but state the exception name directly, so no more colon needed.